src/components/embed/giveGabForm.js

```diff
@@ -36,9 +35,11 @@ const GiveGabForm = ({
 
   useEffect(() => {
     // Information from StoryBlok GiveGabForm Component
-    window.su_trip_id = tripId || '';
+    // TODO: The ciid is subject to change. Please update once the final name has been confirmed
+    window.ciid = tripId || '';
+    window.amt = depositAmount || '';
```

Comment:
```suggestion
    window.su_amt = depositAmount || '';
```

Since we are setting values on the global context, it is preferred to have a namespace or prefix on our variables "just in case"